src/views/no-usage.njk

```diff
@@ -1,17 +1,24 @@
 {% extends "_base.njk" %}
 
 {% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/panel/macro.njk" import govukPanel %}
 
 {% block content %}
-    {{ govukPanel({
-        titleText: "Meat Bait Return complete",
-        html: "Thank you for completing your meat bait return.<br>"
-      }) }}
+
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
```

Comment:
I think this is a little big, `govuk-heading-l` might be a better choice.

https://design-system.service.gov.uk/styles/typography/#headings

> If your service has lots of long form content, use the corresponding size class. For example an h1 should use govuk-heading-xl, an h2 should use govuk-heading-l and so on.
> 
> If your service has lots of question pages, short form content or pages with long headings, start with govuk-heading-l for an h1. But change it if your pages feel unbalanced – semantic and visual hierarchy do not always need to be the same.